ops/pebble.py

```diff
@@ -308,6 +317,82 @@ def __repr__(self):
                 ).format(self=self)
 
 
+class Config:
```

Comment:
Reminder about the conversation we had: I believe things are all properly namespaced not accounting for the module to be used at all times to distinguish, so that importing the specific names inside a different file will preserve context.

This is true for all the type names, but I won't repeat the comment.

Reply:
Per email discussion, I've changed to `API` to `PebbleClient`, but mostly left the other names alone -- I highly doubt they'll be referenced outside of the `pebble` module, so IMO we don't need the `Pebble...` prefix on everything.